app/models/manageiq/providers/openstack/network_manager.rb

```diff
@@ -124,7 +146,7 @@ def create_security_group_queue(userid, options = {})
     }
     queue_opts = {
       :class_name  => self.class.name,
-      :method_name => 'create_security_group',
+      :method_name => 'create_floating_ip',
```

Comment:
This shouldn't be changed, this is security group not floating ip. I know this one slipped through in the original PR :(